src/store.js

```diff
@@ -186,25 +186,35 @@ function EvolvStore(options) {
     configKeyStates.active.clear();
     configKeyStates.entry.clear();
     effectiveGenome = {};
-    Object.keys(results).forEach(function(eid) {
-      const result = results[eid];
-      genomeKeyStates.loaded.forEach(function(key) {
-        const active = !result.disabled.some(function(disabledKey) {
-          return strings.startsWith(key, disabledKey);
-        });
 
-        if (active) {
-          configKeyStates.active.add(key);
-          const entry = result.entry.some(function(entryKey) {
-            return strings.startsWith(key, entryKey);
-          });
+    const resultsMerged = Object.values(results).reduce((acc, result) => {
```

Comment:
Unfortunately `Object.values` is not supported in IE. So you will need to find another way of iterating through the object, or add the method to the objects ponyfill, found under `ponyfills/`